src/audio/codec_adapter/codec_adapter.c

```diff
@@ -238,50 +224,58 @@ int codec_adapter_params(struct comp_dev *dev,
 	return 0;
 }
 
+/**
+ * Function to copy from source buffer to the module buffer
+ * @source: source audio buffer stream
+ * @buff: pointer to the module input buffer
+ * @buff_size: size of the module input buffer
+ * @bytes: number of bytes available in the source buffer
```

Comment:
These are not doxygen commands, they generate warnings when building for not just `include/` but with `INPUT= @top_srcdir@/src/` instead.

Please don't add untested doxygen comments.

```
/home/mherber2/SOF/sof/src/audio/module_adapter/module_adapter.c:384: warning: Found unknown command '\source'
/home/mherber2/SOF/sof/src/audio/module_adapter/module_adapter.c:385: warning: Found unknown command '\buff'
/home/mherber2/SOF/sof/src/audio/module_adapter/module_adapter.c:386: warning: Found unknown command '\buff_size'
/home/mherber2/SOF/sof/src/audio/module_adapter/module_adapter.c:411: warning: Found unknown command '\sink'
/home/mherber2/SOF/sof/src/audio/module_adapter/module_adapter.c:412: warning: Found unknown command '\buff'
/home/mherber2/SOF/sof/src/audio/module_adapter/module_adapter.c:411: warning: Found unknown command '\sink'
/home/mherber2/SOF/sof/src/audio/module_adapter/module_adapter.c:412: warning: Found unknown command '\buff'
/home/mherber2/SOF/sof/src/audio/module_adapter/module_adapter.c:384: warning: Found unknown command '\source'
/home/mherber2/SOF/sof/src/audio/module_adapter/module_adapter.c:385: warning: Found unknown command '\buff'
/home/mherber2/SOF/sof/src/audio/module_adapter/module_adapter.c:386: warning: Found unknown command '\buff_size'
  parameter 'spec'
  parameter 'spec'
/home/mherber2/SOF/sof/src/audio/module_adapter/module/generic.c:335: warning: Found unknown command '\fragment'
/home/mherber2/SOF/sof/src/audio/module_adapter/module/generic.c:336: warning: Found unknown command '\params'
/home/mherber2/SOF/sof/src/audio/module_adapter/module/generic.c:337: warning: Found unknown command '\params'
/home/mherber2/SOF/sof/src/audio/module_adapter/module/generic.c:337: warning: Found unknown command '\response'
```